### PR TITLE
Use Tugboat as a place to run some tests

### PR DESCRIPTION
**Pull request by benjdlambert**
#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)

### CODE REVIEW

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4388-neycbiclogfypkmerjeh0ksngtwv0gbm.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/601c11b414dc58244bc2ae0d

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4388-neycbiclogfypkmerjeh0ksngtwv0gbm.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/601c11b414dc58244bc2ae0d

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4388-neycbiclogfypkmerjeh0ksngtwv0gbm.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/601c11b414dc58244bc2ae0d

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4388-neycbiclogfypkmerjeh0ksngtwv0gbm.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/601c11b414dc58244bc2ae0d

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4388-neycbiclogfypkmerjeh0ksngtwv0gbm.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/601c11b414dc58244bc2ae0d

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4388-neycbiclogfypkmerjeh0ksngtwv0gbm.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/601c11b414dc58244bc2ae0d

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4388-neycbiclogfypkmerjeh0ksngtwv0gbm.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/601c11b414dc58244bc2ae0d

File: .github/workflows/tugboat.yml [hunk 1]
```diff
@@ -48,11 +48,14 @@ jobs:
...
             });
 
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v2
       - uses: actions/setup-node@v1
         with:
           node-version: '14'
 
+      - name: yarn install
+        run: yarn workspace e2e-test install
```
Comment (freben) [inline .github/workflows/tugboat.yml:57]: i'm curious, does the workspace scope make a difference here, when it's in a lerna monorepo?

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4388-neycbiclogfypkmerjeh0ksngtwv0gbm.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/601c11b414dc58244bc2ae0d

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4388-neycbiclogfypkmerjeh0ksngtwv0gbm.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/601c11b414dc58244bc2ae0d

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4388-neycbiclogfypkmerjeh0ksngtwv0gbm.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/601c11b414dc58244bc2ae0d

**Comment by benjdlambert** [comment]
@Rugvip have any ideas how I can get typescript to play nice with both the new types that are pulled in with cypress and jest with our tooling?

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4388-neycbiclogfypkmerjeh0ksngtwv0gbm.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/601c11b414dc58244bc2ae0d

File: .github/workflows/tugboat.yml (hunk 1 above)
Comment (Rugvip) [reply .github/workflows/tugboat.yml:57]: Nop, better use our cached yarn install steps instead

**Comment by Rugvip** [comment]
> @Rugvip have any ideas how I can get typescript to play nice with both the new types that are pulled in with cypress and jest with our tooling?

Are jest and cypress trying to declare the same global symbols? 😅 
Dunno tbh, maybe exclude the cypress things from the main tsconfig, and create a separate one only for e2e tests? There's https://github.com/cypress-io/cypress-and-jest-typescript-example too, so seems like a know issue

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4388-neycbiclogfypkmerjeh0ksngtwv0gbm.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/601c11b414dc58244bc2ae0d

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4388-neycbiclogfypkmerjeh0ksngtwv0gbm.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/601c11b414dc58244bc2ae0d

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4388-neycbiclogfypkmerjeh0ksngtwv0gbm.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/601c11b414dc58244bc2ae0d

**Comment by benjdlambert** [comment]
@Rugvip 🙌

**Review by Rugvip — COMMENTED** [review]
🎉 

A `cypress/README.md` tho please :grin:

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4388-neycbiclogfypkmerjeh0ksngtwv0gbm.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/601c11b414dc58244bc2ae0d

**Comment by benjdlambert** [comment]
@Rugvip +1 me plz?

File: packages/e2e-test/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{
```
Comment (Rugvip) [inline packages/e2e-test/tsconfig.json:1]: Shouldn't need this anymore right?

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4388-neycbiclogfypkmerjeh0ksngtwv0gbm.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/601c11b414dc58244bc2ae0d

File: packages/e2e-test/tsconfig.json (hunk 2 above)
Comment (benjdlambert) [reply packages/e2e-test/tsconfig.json]: done :)

**Review by Rugvip — APPROVED** [review]
:shipit:  😎

**Pull request merged**